### DISCUSSION TITLE
Feature proposal

### DISCUSSION THREAD

**Discussion by ghost** [pyscript/pyscript#1871] [Proposals]
Is it possible to hide "View Code" as code is already seen on the page and "View Code" can be displayed on the site like "Install"(which only displays on site)
<img width="875" alt="19" src="https://github.com/pyscript/pyscript/assets/79505893/0fbb3189-1ed2-4ff3-91cc-1ecefe5db81a">
<img width="953" alt="20" src="https://github.com/pyscript/pyscript/assets/79505893/e53c2fb3-2860-4f4c-87cd-928d8cf94b76">

**Comment by ghost** [comment]
Since no updates.

**Comment by WebReflection** [comment]
@nasrin1748 apologies I've missed this one ... so this is not the place to file these issues as these are PyScript .com related, where none of its code is part of this repository. I might add @FabioRosado in here as it feels a sensible question and yet, where you see that is in an iframe, the page that once runned on full screen provides that functionality ... so what are your arguments to make that page look different in its actual real-world preview? 'cause here I might disagree myself, as that's just n iframe, but maybe you have reasons to make that iframe don't look like the exact page you're going to eventually share?

**Reply by WebReflection** [reply]
P.S. if you are caring about the fact `Install` is visible only on whole site that's because it won't otherwise install that site but the top level PyScript .com site where you are editing code, which is usually undesirable ... but somehow now I see why you think that view code should not be there, as you are actually viewing that code, so maybe it's an easy one to tackle and my previous comment could be ignored.

**Reply by ghost** [reply]
My concern was `view code` can be made visible only on whole site like `Install ` (which is visible only on whole site) becz we don't need `view code` for source1 as it takes us to the same page.
